plugin/src/rules/sort_imports.rs

```diff
@@ -0,0 +1,687 @@
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import b from 'b';import a from 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import b from 'b'; /* comment */ import a from 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import b from 'b'; // comment\nimport a from 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import b from 'b'; // comment 1\n/* comment 2 */import a from 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import { b } from 'b'; /* comment line 1 \n comment line 2 */ import { a } from 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    {
+                        code => "import b\nfrom 'b'; import a\nfrom 'a';",
+                        output => None,
+                        options => { allow_separated_groups => false },
+                        errors => [{
+                            message_id => "sort_imports_alphabetically",
+                            type => ImportStatement
+                        }]
+                    },
+                    // TODO: uncomment these if https://github.com/tree-sitter/tree-sitter-javascript/issues/283 is resolved
```

Comment:
Opened https://github.com/tree-sitter/tree-sitter-javascript/issues/283 since it looks like this unusual line-breaking inside import statements isn't currently parsing correctly